modules/login/php/requestaccount.class.inc

```diff
@@ -211,18 +242,6 @@ class RequestAccount extends \NDB_Form
             }
         }
 
-        // Verify reCAPTCHA
```

Comment:
This is moved to `_validate` instead of `_process`. It doesn't make sense to insert a user into the DB if they failed the reCAPTCHA.